src/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -31,6 +31,15 @@ public static void BufferSize_GettingSameAsWindowSize()
     }
 
     [Fact]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp | TargetFrameworkMonikers.NetcoreUwp)]
+    public static void WindowWidth_WindowHeight_InvalidSize_net46()
+    {
+        Assert.Throws<IOException>(() => Console.WindowWidth = 0);
+        Assert.Throws<IOException>(() => Console.WindowHeight = 0);
+    }
+
+    [Fact]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
@stephentoub @weshaggard do you believe we should have an issue to revert more of these? I imagine it would mean scanning for desktop specific tests that look for an exception type.

Example I noticed recently - this introduced a return null where an exception was thrown by desktop
https://github.com/dotnet/corefx/pull/68/files#diff-78dc0fa34083ddde0ff064b98b058ba5
https://github.com/dotnet/corefx/blob/49e18e5022396a7c31d902bfc65b00fa47e0542e/src/System.Private.Xml/tests/XmlDocument/XmlAttributeCollectionTests/InsertAfterTests.cs#L235